lib/Module/Metadata.pm

```diff
@@ -79,6 +79,15 @@ my $PKG_REGEXP  = qr{   # match a package declaration
   [;\{]                 # semicolon line terminator or block start (since 5.16)
 }x;
 
+my $CLASS_REGEXP = qr{  # match a class declaration from Object::Pad
+  ^[\s\{:]*             # intro chars on a line
```

Comment:
Is this meant to be a `;`?

Reply:
Oops, yes. Should be same as the `package` version earlier